conf/configuration.yaml

```diff
@@ -74,6 +74,7 @@ arlas_auth:
 
 key_header: ${ARLAS_PERSISTENCE_KEY_HEADER:-X-Forwarded-User}
 persistence_engine: ${ARLAS_PERSISTENCE_ENGINE:-hibernate}
+firestore_collection: ${ARLAS_PERSISTENCE_FIRESTORE_COLLECTION:-}
```

Comment:
There is no default value ?

Reply:
No, because we need to ensure that its is chosen to be different than any existing one.

Reply:
Ok